content/doc/book/managing/plugins.adoc

```diff
@@ -225,6 +225,67 @@ The configuration(s) created by the disabled plugin behave as if the plugin
 were uninstalled, insofar that they result in warnings on boot but are
 otherwise ignored.
 
+==== Using the Jenkins CLI
+
+It is also possible to enable and disable plugins using the <<cli#,Jenkins CLI>>.
+Since https://jenkins.io/changelog/#v2.136[Jenkins 2.136], administrators may 
+invoke the `enable-plugin` command to re-enable a collection of plugins which 
+were previously disabled and since https://jenkins.io/changelog/#v2.151[Jenkins 2.151] 
+the `disable-plugin`, which provides a command to disable installed plugins, is included.
+
+The `enable-plugin` command receives a list of plugins to be enabled. If any of 
+the plugins have one or more dependencies, these plugins and their dependencies
+will be enabled.
+
+[source]
+----
+java -jar jenkins-cli.jar -s http://localhost:8080/ enable-plugin PLUGIN ... [-restart]
+
+Enables one or more installed plugins transitively.
+
+ PLUGIN   : Enables the plugins with the given short names and their
+            dependencies.
+ -restart : Restart Jenkins after enabling plugins.
+----
+
+The `disable-plugin` command receives a list of plugins to be disabled. The 
+output will display messages for both successful and failed operations. In case
+that only the error messages are desired, the `-quiet` option can be specified.
+How to deal with other plugins which depend on one or more of the plugins 
+passed as parameter may be specified with the `-strategy` option.
+
+[source]
+----
+java -jar jenkins-cli.jar -s http://localhost:8080/ disable-plugin PLUGIN ... [-quiet (-q)]
+[-restart (-r)] [-strategy (-s) strategy]
+
+Disable one or more installed plugins.
+Disable the plugins with the given short names. You can define how to proceed with the 
+dependant plugins and if a restart after should be done. You can also set the quiet mode 
```

Comment:
"dependant" should go to "dependent" in all instances in this document.

(Discussion here on how the British use the two different spellings and Americans only the "dependent" form: https://www.k12reader.com/term/dependent-vs-dependant/ )

Reply:
This is a copy of the CLI right?  So, again out of scope for this change.